doc/akri_connector/connector-metadata-schema.json

```diff
@@ -0,0 +1,328 @@
+    "type": "object",
+    "description": "Details about the source code that compiled this connector",
+    "properties": {
+      "language": {
+        "type": "string",
+        "description": "The programming language used to write this connector"
+      },
+      "languageVersion": {
+        "type": "string",
+        "description": "The version of the programming language used to write this connector"
+      },
+      "sdks": {
+        "type": "object",
+        "description": "Details about the versions of the AIO SDKs used to write this connector (if they were used)",
+        "properties": {
+          "mqttPackageVersion": {
+            "type": "string",
+            "description": "The version of the Azure IoT Operations MQTT package"
+          },
+          "servicesPackageVersion": {
+            "type": "string",
+            "description": "The version of the Azure IoT Operations services package"
+          },
+          "connectorPackageVersion": {
+            "type": "string",
+            "description": "The version of the Azure IoT Operations connector package"
+          }
+        }
+      }
+    },
+    "required": [
+      "language",
+      "languageVersion"
+    ]
+  },
+  "aioMetadata": {
+    "type": "object",
+    "description": "The AIO requirements to run this connector",
+    "properties": {
+      "aioMinVersion": {
+        "type": "string",
+        "description": "The minimal (inclusive) version that can run this connector."
+      },
+      "aioMaxVersion": {
+        "type": "string",
+        "description": "The maximal (inclusive) version that can run this connector."
+      }
+    }
+  },
+  "inboundEndpoints": {
+    "type": "array",
+    "description": "The array of inbound endpoints in a device that this connector will be deployed for.",
+    "items": [
+      {
+        "type": "object",
+        "properties": {
+          "endpointType": {
+            "type": "string",
+            "description": "The type of this inbound endpoint."
+          },
+          "configSchemas": {
+            "type": "object",
+            "description": "The inbound endpoint level JSON schemas for the various 'additional configuration' fields. A JSON schema must be provided for each 'additional configuration' only if that field is used by this connector.",
+            "properties": {
+              "additionalConfigurationSchema": {
+                "$ref": "#/definitions/schema"
+              }
+            }
+          },
+          "assets": {
+            "type": "array",
+            "description": "The array of assets that reference this inbound endpoint",
+            "items": [
+              {
+                "type": "object",
+                "properties": {
+                  "assetName": {
+                    "type": "string",
+                    "description": "The name of the asset"
+                  },
+                  "configSchemas": {
+                    "type": "object",
```

Comment:
The Asset CRD (see PR here: ([https://msazure.visualstudio.com/One/_git/Azure-IoT-Platform-DeviceRegistry/pullrequest/11879236](https://msazure.visualstudio.com/One/_git/Azure-IoT-Platform-DeviceRegistry/pullrequest/11879236)) contains the following schema fields:

- datasetConfiguration
- datasetDataPointConfiguration
- eventConfiguration
- eventDataPointConfiguration
- streamConfiguration
- managementGroupConfiguration
- actionConfiguration

Just add Schema at the end of each and then we are in sync with Asset CRD.

cc: @varunpuranik